techLogger.js

```diff
@@ -125,58 +162,75 @@ function _log(level, log) {
 }
 
 /**
+ * Returns the configuration of the logger
+ *
+ * @return {Object} the configuration of the logger
+ */
+function _getLoggerConfig() {
+    return global.NODE_TECH_LOGGER_CFG;
+}
+
+/**
+ * Set the configuration of the logger
+ *
+ * @param  {Object} config the logger configuration
+ */
+function _setLoggerConfig(config) {
+    global.NODE_TECH_LOGGER_CFG = {
+        date: new Date().getTime(),
+        config: config
+    };
+}
+
+/**
  * Initialize winston logger
  */
-function _initLogger() {
-    var config = global.NODE_TECH_LOGGER_CFG;
-    if (!config) {
+function _configureLogger() {
+    var loggerConfig = _getLoggerConfig();
+    if (!loggerConfig) {
         // Default config
-        config = {};
-        global.NODE_TECH_LOGGER_CFG = config;
+        _setLoggerConfig({});
+        loggerConfig = _getLoggerConfig();
 
         var msg = "\n\n!!! =========================================== !!!";
         msg += "\n!!!  No configuration set for node-tech-logger  !!!";
         msg += "\n!!! =========================================== !!!\n\n";
         console.warn(msg);
     }
+    console.info("Logger has been configured");
```

Comment:
At that time, the loggers have not been configured yet :stuck_out_tongue_winking_eye:
Shouldn't this be moved at the end of the function?